### PR TITLE
Add sort instructions for Windhelm Smelterworks

### PR DESCRIPTION
**Pull request by cecell**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -19040,3 +19040,11 @@ plugins:
...
           - '[Perks Unbound SSE -- WACCF patch](https://www.nexusmods.com/skyrimspecialedition/mods/46295)'
         condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and not active("PerksUnbound_WACCF.esp")'
+        
+  ### Windhelm Smelterworks ###
+  - name: 'Windhelm Smelterworks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/46333'
+        name: 'Windhelm Smelterworks'
+    after:
+      - 'Windhelm Exterior Altered.esp'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
    after: [ 'Windhelm Exterior Altered.esp' ]
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@cecell Can you add a message for the patch between the two mods?

```yaml
    msg:
      - <<: *patchProvided
        subs: [ 'Windhelm Exterior Altered' ]
        condition: 'active("Windhelm Exterior Altered.esp") and not active("Smelterworks - Windhelm Ext Alt Patch.esp")'
```

Also, can you add the cleaning data?

```yaml
    dirty:
      - <<: *reqManualFix
        crc: 0xAEBCDCBF
        util: '[SSEEdit v4.0.3f](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 38
        udr: 1
        nav: 3
      - <<: *reqManualFix
        crc: 0x658B58CA
        util: '[SSEEdit v4.0.3f](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        nav: 3
```

**Comment by cecell** [comment]
Sure what you types in is great but I don't know how to make more changes to my commit and I don't see an button to commit your changes.  Your changes look great though

**Comment by sibir-ine** [comment]
> 
> 
> Sure what you types in is great but I don't know how to make more changes to my commit and I don't see an button to commit your changes.

@cecell You can just make the changes on your local branch & push them; they'll show up here.

**Review by sibir-ine — APPROVED** [review]

**Comment by sibir-ine** [comment]
@cecell Thanks!

**Pull request merged**